modules/ai/assets/js/editor/components/prompt-history/parts/modal-item.js

```diff
@@ -0,0 +1,150 @@
+import {
+	Box,
+	ListItem,
+	ListItemButton,
+	ListItemIcon,
+	ListItemText,
+	Stack,
+	styled,
+} from '@elementor/ui';
+import EditIcon from '../../../icons/edit-icon';
+import { TrashIcon } from '@elementor/icons';
+import RestoreIcon from '../../../icons/restore-icon';
+import ActionButton from './modal-item-action-button';
+import { ACTION_TYPES, usePromptHistoryAction } from '../context/prompt-history-action-context';
+import PromptHistoryActionIcon from './action-icon';
+import RefreshIcon from '../../../icons/refresh-icon';
+import PromptHistoryItemTitle from './modal-item-title';
+import PromptHistoryItemSecondaryContent from './modal-item-secondary-content';
+
+const StyledListItem = styled( ListItem )`
+  & .e-prompt-history-item-actions {
```

Comment:
Please fix indentations here and all the below CSS.

Reply:
I think I don't get it 🙁 

<img width="834" alt="elementor-pro_–_modal-item_js" src="https://github.com/elementor/elementor/assets/139438463/348fc812-a6aa-498b-893c-711fbeca3f3b">